src/errors/runtime/message.js

```diff
@@ -24,6 +24,8 @@ export default {
     testNameIsNotAString:                           'The test name is expected to be a string, but it was {type}.',
     testBodyIsNotAFunction:                         'The test body is expected to be a function, but it was {type}.',
     beforeEachIsNotAFunction:                       `Fixture's "beforeEach" method takes a function, but {type} was passed.`,
+    invalidAuthCredentialsFormat:                   `Fixture's "httpAuth" method accepts a credentials object with the "username" and "password" fields.`,
```

Comment:
Fixture's "httpAuth" method accepts a credentials object with the "username" and "password" **string** fields. ?